### PR TITLE
feat(protocol-designer): replace computing overlay with wait cursor

### PR DESCRIPTION
**Pull request by IanLondon**
# Overview

Alternative UX vs the loading spinner overlay. In this version, there is no spinner overlay, instead there is an invisible overlay (to block user interaction while computation is happening) and the cursor changes to "wait" until computation is complete.

# Changelog

- Replace (unreleased) overlay spinner with an invisible overlay and `cursor: wait`

# Review requests

- [ ] Make 100 mixes or something to give you a few seconds of computation
- [ ] Cursor should change to "wait" icon (I think it's OS-dependent, for me it's a small gray spinner)
- [ ] Cursor is still "wait" no matter where your mouse is in PD (eg, over the sidebar, nav bar, etc)
- [ ] While cursor is "wait", all clicks should be ignored

# Risk assessment

Low, style change only

### CODE REVIEW

**Review by shlokamin — COMMENTED** [review]
This is a huge step up from the `screen is frozen and it looks like nothing is happening` experience that we currently have, while being less `in your face` than the spinner. 

My only question — should this fall under `TYPOGRAPHY_PROPS`? Or should we like, create a new const (if design approves)?

File: protocol-designer/src/components/ComputingSpinner.js [hunk 1]
```diff
@@ -1,34 +1,25 @@
...
   const showSpinner = useSelector(fileDataSelectors.getTimelineIsBeingComputed)
 
   return (
     showSpinner && (
-      <Flex
-        alignItems={ALIGN_CENTER}
-        justifyContent={JUSTIFY_CENTER}
-        backgroundColor="rgba(115, 115, 115, 0.9)"
+      <Box
+        cursor="wait" // TODO(IL, 2020-07-16): make a const CURSOR_WAIT to import here
```
Comment (mcous) [inline protocol-designer/src/components/ComputingSpinner.js:13]: Forgot to talk about this in slack before we threw up our hands and said "typography", but `cursor` is a relatively rare (I think?) thing to set, so we could choose to bail out to CSS here:

```js
<Box
  ...
  css={css`
    cursor: wait;
  `}
/>
```

File: components/src/primitives/style-props.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ import type { Styles } from 'styled-components'
 const COLOR_PROPS = ['color', 'backgroundColor', 'opacity']
 
 const TYPOGRAPHY_PROPS = [
+  'cursor',
```
Comment (mcous) [inline components/src/primitives/style-props.js:14]: If we're gonna add this style prop, we need to make sure documentation (`components/src/primitives/README.md`) is updated

Comment (IanLondon) [reply components/src/primitives/style-props.js:14]: and `primitives/types.js`?

**Review by shlokamin — APPROVED** [review]
LGTM

**Review by Kadee80 — APPROVED** [review]
🍝

**Pull request merged**